Lesson04_Task01.py

```diff
@@ -0,0 +1,20 @@
+"""
+Lesson04 Task01.
+
+# we have four values w,x,y,z
+# write if-elif-else statement that will search minimum value
+and print smth aka "'y' is minimum value'
+# advice use python debugger and different values to check your algorithm
+w, x, y, z = 100, 200, 40, 300
+"""
+
+w, x, y, z = 100, 200, 40, 300
+
+if w <= x and w <= y and w <= z:
+    print("'w' is the minimum value")
```

Comment:
In print we have info duplication as one of solutions we could try use templates
TEMPLATE = '{} is minimum value'
print(TEMPLATE.format('w'))

But the best way - store found value to new variable and then do just one print at the end of all